### DISCUSSION TITLE
Bug: First repo can be added to a list in an unstarred state

### DISCUSSION THREAD

**Discussion by 01231** [community/community#10262] [Other Feature Feedback, Questions, & Ideas] [answered]
I can add some repos to a list without having to star them, but others automatically receive a star when I add them to a list

So I'm not quiet sure where the bug is:
- one shouldn't be allowed to add repos to a list without having to star them or
- it's already possible to add repos to a list without having to star them, but it doesn't work

How does your GitHub behave?

**Comment by bailey** [comment, marked as answer]
Hi there @noahliechti, you should not be able to add a repo to a list if the repo is not starred itself. When you use the star button dropdown, adding the repo to a list in that dropdown should automatically star it as well.

Could you provide more detail on where you're seeing a repo being added to a list without being starred? Thanks!

**Reply by BenjaminMichaelis** [reply]
I found this issue as well. Hopefully this helps.
Also having it not star at first inspired the desire for #10272 to be implemented for me at least.

https://user-images.githubusercontent.com/22186029/150289439-52e46c6b-ddae-47ec-8ffd-2f92deaaae19.mp4

**Reply by theKAKAN** [reply]
It happens to me as well, but as said above, only when creating the list!

**Comment by 01231** [comment]
@bailey I managed to add repos without starring when I use the dropdown to create a **new** list. The repos is added to the newly created list. Even when I add additional repos to this new list the inital repo stays unstarred.

So adding a repo to a list without having to star it only works on the first repo of a list. Hope this helps.

IMHO I would like to see a feature where repos can be added to lists without starring them.
